src/platform/platform_linux.cpp

```diff
@@ -471,11 +463,12 @@ std::string multipass::platform::host_version()
                : fmt::format("{}-{}", QSysInfo::productType(), QSysInfo::productVersion());
 }
 
-std::filesystem::path mp::platform::Platform::get_root_cert_path() const
+std::filesystem::path mp::platform::Platform::get_root_cert_dir() const
 {
-    constexpr auto* root_cert_file_name = "multipass_root_cert.pem";
-    return mp::utils::in_multipass_snap()
-               ? multipass_final_storage_location() / "data" / daemon_name / "certificates" /
-                     root_cert_file_name
-               : std::filesystem::path{"/usr/local/share/ca-certificates"} / root_cert_file_name;
+    using Path = std::filesystem::path;
+    const auto base_dir = utils::in_multipass_snap()
+                              ? Path{utils::snap_common_dir().toStdString()} / "data"
+                              : Path{"/home/root/.local/share"};
```

Comment:
/home/root ?

Reply:
Also, I wonder if the client user would be able to access the /root/.local/

Reply:
haha whoops, although that's probably why I didn't have permission issues...
But yeah, I don't think the client should (or would) be able to access `/root/.local/share`

Reply:
moved to /usr/local/etc